### ISSUE TITLE
CP Will not manage mapped or network shared drives

### ISSUE THREAD

**Issue by Jakesparrow** [CouchPotato/CouchPotatoServer#6497]
### Steps to reproduce:
1. .. Have a Network Share with movies
2. .. Add the share (\\NAS-1\Movies) or mapped network drive (Z:\Movies) as a movie folder to manage.

### Information:
Movie(s) I have this with: ...1,000's
Quality of the movie being searched: ...N/A
Providers I use: ...UseNet, Torrent
Version of CouchPotato: ... 3.0.1
Running on: ... Windows Server 2012 R2

### Logs:
```
06-13 05:46:33
ERROR
[otato.core.plugins.manage] Directory doesn't exist: X:\Children's Movies
06-13 05:46:33
INFO
[otato.core.plugins.manage] Updating manage library: \\NAS-1\Media\Movies
```

I have 4 drives at the moment that movies are stored in. 2 on board drives, D:\ and F:\ that are managed fine, and 2 NAS'. To test both mapping and network share path, I have both at the moment. The logs reflect both as \\NAS-1\Movies and X:\Children's Movies.
Although it says it is updating NAS-1, it never locates anything, and the percentage stays at 0%, and nothing else is ever reporting in logs.

**Comment by CouchPotatoBot** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Fill in all the dots. They're there for a reason
- [ ] Post logs, either inline (for smaller logs) or using [Pastebin](http://pastebin.com/)
- [ ] Post full version information

Please make sure you also read [contribution guide](https://github.com/RuudBurger/CouchPotatoServer/blob/develop/contributing.md#issues) and followed all the steps. 
Make the title describe your issue. Having "CP not working" or "I get this bug" for 100 issues, isn't really helpful. My master will close issues if there isn't enough information. On a good day he will tag the issue on close with the reason (like `can't reproduce`), but usually he won't, the lazy asshat.

Sometimes my master seems like a grumpy cat and responds with short answers. This isn't (always) because he hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let him know. Maybe he can teach me his awesome ways.

Thanks!

**Comment by Jakesparrow** [comment]
Also, I have tried running CP as a service as well. No difference that I could tell.

**Comment by blade316** [comment]
@Jakesparrow CP doesnt play well with UNC or mapped drives .... Please confirm:

1. are you running CP on the Windows Server 2012 R2 box?
2. what NAS device are you running?
3. Does your NAS support having CP installed on it?

Also have you tried using a symbolic link? ..... You can create one using the MKLINK command

mklink /D "C:\LinkName" "\NetworkLocation\LocationName"

http://www.howtogeek.com/howto/16226/complete-guide-to-symbolic-links-symlinks-on-windows-or-linux/

**Comment by Jakesparrow** [comment]
1. Yes
2. WD My Book Live NAS'
3. The NAS does not have anything installed on it. It is only storage with Samaba shares.

I have since got it working, and not sure what the solution eventually was, but here were my steps to resolve:

I uninstalled CP, and did a regedit to remove all pertaining entries.
I downloaded latest release and reconfigured with my previous settings.
I tried to add the NAS' again using their network shares. No success. I added additional backslashes. Also no success. So I mapped the drives and added and it started working. 
Not the end of the world I suppose, although I dislike using mapped drives on a headless server.

I have not tried a symbolic link. Might try that later.

Thanks for the response.
